pytests/conftest.py

```diff
@@ -47,11 +69,21 @@ def check_package(self, package):
                 return True
         return False
 
-    def remove_package(self, package):
-        ret = self.run([ 'tdnf', 'erase', '-y', package ])
-        if ret['retval'] != 0:
-            return False
-        return True
+    def erase_package(self, pkgname, pkgversion=None):
+        if pkgversion:
+            pkg = pkgname + '-' + pkgversion
+        else:
+            pkg = pkgname
+        self.run([ 'tdnf', 'erase', '-y', pkg ])
+        assert(self.check_package(pkgname) == False)
+
+    def install_package(utils, pkgname, pkgversion=None):
+        if pkgversion:
+            pkg = pkgname + '-' + pkgversion
+        else:
+            pkg = pkgname
+        utils.run([ 'tdnf', 'install', '-y', '--nogpgcheck', pkg ])
```

Comment:
I see that by default "--nogpgcheck" is added for all installs. Did we have test case scenario which tests gpgcheck?

Reply:
we have a requirement to sign and use a local gpgkey to verify. we should add this with the external gpgkey change. good for now

Reply:
No GPG checks have not been added yet. I believe it was not part of the initial test tuit (in tdnf as well). If they were done as a part of harness tests, then I will be migrated in another PR.

Reply:
Earlier test does not use --nogpgcheck in the command line. Instead it uses both local and photon repos and use the .repo file to decide whether gpgcheck has to be enabled for that repo using gpgcheck=1 inside /etc/yum.repos.d/*.repo. All photon repos are defaulted to gpgcheck enabled and test repo is non gpgchecked by using gpgcheck=0 in repo file.

Reply:
Got it. Thanks. When I go over the gpg section of tests (in another PR), I will handle this correctly.